src/components/UI/JSONSchemaForm/test.schema.json

```
@@ -67,6 +67,81 @@
             "title": "Boolean fields",
             "type": "object"
         },
+        "logic": {
+            "description": "Uses of 'anyOf', 'oneOf', and 'not'.",
+            "properties": {
+                "anyOf": {
+                    "properties": {
+                        "anyOfDeprecated": {
+                            "anyOf": [
+                                {
+                                    "deprecated": true,
+                                    "minLength": 3,
+                                    "type": "string"
+                                },
+                                {
+                                    "minimum": 3,
+                                    "type": "number"
+                                }
+                            ],
+                            "description": "Only the second declared subschema (number, minimum=3) should be visible.",
+                            "title": "Property with subschemas using 'anyOf' and 'deprecated'"
+                        },
+                        "anyOfSimple": {
+                            "anyOf": [
+                                {
+                                    "minLength": 3,
+                                    "type": "string"
+                                },
+                                {
+                                    "minimum": 3,
+                                    "type": "number"
+                                }
+                            ],
+                            "description": "Only the first declared subschema (string, minLenght=3) should be visible.",
```

```suggestion
                            "description": "Only the first declared subschema (string, minLength=3) should be visible.",
```
Small typo, shouldn't block merging